index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
I would love to do this programatically based on the dependancies in the `package.json`, but that can simply be a TODO :smile: